### DISCUSSION TITLE
false bugs

### DISCUSSION THREAD

**Discussion by SpareEnderboy** [FunkinCrew/Funkin#2599] [General]
i've came across a false bug recently
my friday night funking was muted and i thought it was a bug since i rarely adjust the in-game volume

**Comment by NotHyper-474** [comment]
I have a theory that the save migration system might be incorrectly converting either the volume or mute state data, but this would only be the case if you played an earlier version before playing in v0.3.x.

**Reply by NotHyper-474** [reply]
Hmm I deleted my current save data, played on v0.2.7.1, messed around with the volume and mute, went back to v0.3.3 but it didn't start muted (in fact, it started with the same settings I had put in 0.2.7.1) so perhaps my theory is wrong.
